### ISSUE TITLE
Promotion Edge Case

### ISSUE THREAD

**Issue by McTsts** [McTsts/Werewolves-Roles#758]
For bigger games: If there are both a Flute Charmer and Flute Apprentice and the Flute Player dies, do both promote into a Flute Player, or does only one do so (and if this is the case, which one does).

_Originally posted by @McTsts in https://github.com/McTsts/Werewolves-Roles/issues/755#issuecomment-1264456846_
      
In general there is no proper solution for several roles promoting into the same role. The current roles would imply all roles that may promote promote into it at the same time.

**Comment by 12Metatron** [comment]
this 'solution' holds true with promotion roles from other solo teams, such as wraith. Though I think its most relevant for this one

**Comment by McTsts** [comment]
Two FPAs promoting into a FP would be a bit broken compared to two Wraiths gaining the sleepless power I think.

**Comment by 12Metatron** [comment]
yeah. and i think with wisp and corpse, this current solution works for them

**Comment by McTsts** [comment]
So I think everyone just promoting at the same time is fine tbh and this doesn't require any changes.
